README.md

```diff
@@ -9,3 +9,37 @@ Run as follows:
 ```
 $ cargo build --release && cargo run --release --bin vtable_bench
 ```
+
+
+# Why this is worth measuring
+
+Trait objects' vtables are passed around as fat pointers. If they're used
+rarely, this seems unlikely to make much performance difference either way.
+However, if we have lots of trait objects, and if we're expecting to have a
+reasonably large number of trait objects, and for many of those to be multiply
+aliased, it's not obvious if fat pointers are a better trade-off vs. storing the
+vtable pointer in the object itself.
+
+These benchmarks are an attempt to start understanding the performance
+trade-offs. They are highly simplistic, and inevitably only tell us about a
+small handful of possible performance points. They might be better than
+nothing, however.
+
+Each benchmark creates one or more trait objects, puts them in a vector, and
+then iterates over that vector calling a method:
+
+* The `normal` benchmarks use fat pointers and the `alongside` benchmarks store
+  the vtable alongside the data itself (recreating fat pointers on an as-needed
+  basis).
+
+* The `multiref` benchmarks allocate a single box and multiply alias it;
+  non-`multiref` benchmarks allocate multiple boxes without allocation.
```

Comment:
I still don't understand this sentence. It allocates without allocating?

Reply:
Should be "without aliasing".